pkg/roachpb/api.proto

```diff
@@ -1464,13 +1464,6 @@ message ComputeChecksumResponse {
       (gogoproto.customtype) = "github.com/cockroachdb/cockroach/pkg/util/uuid.UUID"];
 }
 
-// RetryTracingEvent is the trace recording used to track retries.
-message RetryTracingEvent {
-  string operation = 1;
-  int32 attempt_number = 2;
-  string retry_error = 3;
-}
```

Comment:
I was concerned that it wasn't safe to remove this message type in the case of importing a remote recording from a node on an older version, but it looks to me like we ignore unmarshalling errors, at least when sending them to listeners.

Reply:
Very good point but yeah https://github.com/cockroachdb/cockroach/blob/2d291751692bb40cf634504c74235ff25d1c80aa/pkg/util/tracing/crdbspan.go#L814 looks like we don't error out if unmarshalling failed. I could keep the proto around and just stop sending it for a release out of caution.

Reply:
added the proto back, will delete it in 23.2